index.js

```diff
@@ -0,0 +1,8 @@
+const http = require('http')
+
+http.createServer(function (request, response) {
```

Comment:
you can make it an arrow function as you did everywhere

Reply:
This file is removed in the next PRs, so I will not change it here.